File: app/src/main/java/com/example/ecnill/separatedlogin/Utils/FragmentChangeListener.java
package com.example.ecnill.separatedlogin.Utils;

import android.support.v4.app.Fragment;

/**
 * Created by ostr on 27/02/17.
 */

public interface FragmentChangeListener {

    void replaceFragment(int containerId, Fragment fragment, boolean addToBackStack);

}

File: app/src/main/java/com/example/ecnill/separatedlogin/utils/FragmentChangeListener.kt
package com.example.ecnill.separatedlogin.utils

import android.support.v4.app.Fragment

/**
 * Created by ecnill on 27/02/17.
 */

interface FragmentChangeListener {

    fun replaceFragment(containerId: Int, fragment: Fragment, addToBackStack: Boolean)

}
